src/traces/isosurface/attributes.js

```diff
@@ -235,10 +235,11 @@ colorscaleAttrs('', {
 
     colorbar: colorbarAttrs,
 
-    contour: meshAttrs.contour,
+    opacity: meshAttrs.opacity,
```

Comment:
We should probably point out in the mesh3d attributes that the current behavior isn't perfect and may be subject to change in the (near) future cc https://github.com/plotly/plotly.js/issues/3243